### PR TITLE
Add committee page

### PR DESCRIPTION
**Pull request by Terbau**
## Changelog

- Adds a page to display info about a committee. Uses the same layout as the company page
- Extends the committee table with additional fields such as `description`, `email` and `image`

## How to test

1. Run migrations (I recommend `pnpm migrate-with-fixtures` for the latest fixture data)
2. Spin monoweb up and go to `/committee`

## Preview
image

### CODE REVIEW

File: apps/web/src/components/layout/EntryDetailLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+export interface EntryDetailLayoutProps {
+  title: string
+  type?: string
+  color?: "BLUE" | "GREEN" | "AMBER"
+}
+
+export const EntryDetailLayout: FC<PropsWithChildren<EntryDetailLayoutProps>> = ({ children, title, type, color }) => {
+  const borderColorClass = color ? `border-${color.toLowerCase()}-8` : "border-blue-8"
+  const textColorClass = color ? `text-${color.toLowerCase()}-11` : "text-blue-11"
```
Comment (matsjla) [inline apps/web/src/components/layout/EntryDetailLayout.tsx:12]: I don't think tailwind is clever enough to strip this at compile time, should do something like
```suggestion
export const EntryDetailLayout: FC<PropsWithChildren<EntryDetailLayoutProps>> = ({ children, title, type, color = 'BLUE' }) => {
  const borderColorClass = cn({
    "border-blue-8": color === 'BLUE',
    "border-green-8": color === 'GREEN',
    "border-amber-8": color === 'AMBER',
  })
  // same for textColorClass
```
})

File: packages/config/tailwind-preset.js [hunk 2]
```diff
@@ -82,6 +82,14 @@ module.exports = {
...
     require("tailwindcss-animate"),
   ],
+  safelist: [
+    {
+      pattern: /text-(blue|amber|red|green|indigo)-11/,
+    },
+    {
+      pattern: /border-(blue|amber|red|green|indigo)-8/,
+    },
+  ],
```
Comment (matsjla) [inline packages/config/tailwind-preset.js:92]: I think it's better to describe all the classnames that will be used than maintaining the list here

File: apps/sanity/package.json [hunk 3]
```diff
@@ -19,7 +19,8 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-is": "^18.2.0",
-    "sanity": "^3.0.0"
+    "sanity": "^3.0.0",
+    "styled-components": "^5.2"
```
[the remark this answers is not in the document]
Comment (Terbau) [reply apps/sanity/package.json:23]: Can't be bothered to fix this anymore. Problem for a another time Cx

File: apps/web/src/components/layout/EntryDetailLayout.tsx (hunk 1 above)
Comment (Terbau) [reply apps/web/src/components/layout/EntryDetailLayout.tsx:12]: In my experience this should work, but I agree. Its harder to maintain without knowing exactly how tailwind works, and its way more clear to use your suggested and more intuitive method. I'll get it sorted.

**Pull request merged**